### DISCUSSION TITLE
Heap Inspection (CWE-244)

### DISCUSSION THREAD

**Discussion by furkanckmk** [coturn/coturn#1353] [General]
Vulnerability Severity: Low
Impact of Exploitation: Unauthorized Access to Sensitive Information, Privacy Violation

File Name - Line Number
\coturn\src\apps\relay\dbdrivers\dbd_mysql.c - 46, 55
\coturn\src\apps\relay\dbdrivers\dbd_redis.c - 52
\coturn\src\apps\relay\ns_ioalib_engine_impl.c - 3567

Please review the document for more detailed information.
[COTURN_heap-inspection.pdf](https://github.com/coturn/coturn/files/13701267/COTURN_heap-inspection.pdf)

**Comment by eakraly** [comment]
Hi @furkanckmk 
Please open an issue and provide all details there, not in PDF format.
This is a discussion forum. Thank you.

**Reply by furkanckmk** [reply]
The issue was first shared here to attract attention. It was prepared as a document for understanding detailed information. Therefore, it is a PDF and you should not have any problems opening it.
